_i18n/de/resources/moneropedia/wallet.md

```diff
@@ -1,18 +1,17 @@
 ---
-terms: ["wallet", "wallets"]
-summary: "A wallet stores the information necessary to send and receive Monero"
+terms: ["wallet", "wallets", "Wallet", "Wallets"]
+summary: "Ein Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A Monero account, or wallet, stores the information necessary to send and receive Moneroj.  In addition to sending and receiving, the Monero Wallet software keeps a private history of your transactions and allows you to cryptographically sign messages.  It also includes Monero mining software and an address book.
+Ein Monero-Konto beziehungsweise ein Wallet speichert die Informationen, die zum Senden und Empfangen von Monero benötigt werden. Zusätzlich zum Senden und Empfangen führt und bewahrt die Wallet-Software von Monero eine private Chronik deiner Transaktionen und ermöglicht es dir zudem, Nachrichten kryptografisch zu signieren. Sie enthält außerdem ein Programm zum Minen von Monero und ein Adressbuch.
```

Comment:
@erciccione it says "A monero wallet" but I think this only applies to "the official Monero wallet" - other wallets usually don't come with mining software and message signing capabilities?

Reply:
This entry is referring specifically to the 'official' Wallet, you are right, as hinted by "the Monero Wallet software".